src/core/services/interimElement/interimElement.spec.js

```diff
@@ -263,6 +263,8 @@ describe('$$interimElement service', function() {
 
         Service = $$interimElement();
 
+        ieShow = Service.show;
```

Comment:
FWIW, I think it's not a good idea to mock services methods in tests only for convenience, as it might hide actual issues (or make breakages hard to debug). I would create helper functions instead (e.g. `showAndFlush()`) and call those when necessary. This makes it much easier to understand what is happening in each test (e.g. "here we are calling `.show()` and the `flush()`").

When tinkering with this locally, I used the following helpers:

```js
function callMethodAndFlush(method, args) {
  var result = Service[method].apply(Service, args);
  $material.flushInterimElement();

  return result;
}

function cancelAndFlush() {
  return callMethodAndFlush('cancel', arguments);
}

function hideAndFlush() {
  return callMethodAndFlush('hide', arguments);
}

function showAndFlush() {
  return callMethodAndFlush('show', arguments);
}
```